lib/logic.rb

```diff
@@ -9,28 +10,28 @@ def check_win(mark)
     ver3 = [@positions[2], @positions[5], @positions[8]].all?(mark)
     dia1 = [@positions[0], @positions[4], @positions[8]].all?(mark)
     dia2 = [@positions[2], @positions[4], @positions[6]].all?(mark)
-    result = [hor1, hor2, hor3, ver1, ver2, ver3, dia1, dia2].any?(true)
-    result
+    [hor1, hor2, hor3, ver1, ver2, ver3, dia1, dia2].any?(true)
   end
 
   def check_the_winer(player)
-    msg = "congratulation #{player} you won the game"
-    msg
+    "congratulation #{player} you won the game"
   end
 
   def check_draw(mark)
     @positions.all?(String) and check_win(mark) == false
   end
 end
+
 class CheckPosition
   def check_input(input)
     true if input.to_i.between?(1, 9)
   end
 
   def check_position(position)
-    true if position.class == Integer
+    true if position.instance_of?(Integer)
   end
 end
+
 class Board
   def draw_board(positions)
```

Comment:
I know this is supposed to be for the previous project with tic tac toe implementation, but you really ought to be returning the board as opposed to printing it here. `puts` should only really be in main.rb
It would also make testing this method easier.

[OPTIONAL]